unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs

```diff
@@ -15,11 +16,16 @@ public class ExploreV2MenuComponentController : IExploreV2MenuComponentControlle
     internal IExploreV2MenuComponentView view;
     internal IPlacesAndEventsSectionComponentController placesAndEventsSectionController;
     internal InputAction_Trigger toggleExploreTrigger;
+    internal IExploreV2Analytics exploreV2Analytics;
+    internal ExploreSection currentOpenSection;
+    internal float lastTimeSectionWasOpen = 0f;
+    internal bool anyActionExecutedFromLastOpen = false;
```

Comment:
We should try to avoid adding analytics specific code if possible.  Take a look at `QuestUIAnalytics` and how things like times are managed (`questLogSetVisibleTimeStamp`).

Reply:
Thanks for the suggestion Alex! I have implemented the times magement in the same way you did. Please, take a look!